File: Selenium/Activities/Activity1.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service = service) as driver:
  
  # Open the browser to the URL
  driver.get("https://training-support.net")

  #  Get the title of the page and print it to the console
  print("Home page title:"+driver.title)
  #Find the "About Us" button on the page using it's id. and click on it
  driver.find_element(By.ID,"about-link").click()

  #Print the title of the new page
  print("New page title is: ", driver.title) 

  # Close the browser
  driver.quit()

File: Selenium/Activities/Activity1.java
import org.openqa.selenium.By;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.WebDriver;
import io.github.bonigarcia.wdm.WebDriverManager;
public class Activity1 {
    public static void main(String[] args) {
        // Setup the Firefox driver(GeckoDriver)
        WebDriverManager.firefoxdriver().setup();

        // Create a new instance of the Firefox driver
        WebDriver driver = new FirefoxDriver();

        // Open the browser
        driver.get("https://training-support.net");

        // Get the title of the page and print it to the console
        System.out.println("Home page title: " +driver.getTitle());

        //Find the "About Us" button on the page using it's id. and click on it
        driver.findElement(By.id("about-link")).click();
        //Get the title of the new page and print it to the console.
        System.out.println("Title of the new page is:"+driver.getTitle());

        // Close the browser
        driver.quit();
    }

}
